src/main/java/de/tum/in/www1/artemis/service/LearningPathService.java

```diff
@@ -53,6 +58,19 @@ public class LearningPathService {
 
     private final CompetencyRelationRepository competencyRelationRepository;
 
+    private final static double DUE_DATE_UTILITY = 10;
```

Comment:
Please add comments what these constants mean and how they are used ("due date utility" is not quite understandable for me).

Also use `static final` instead of `final static`.